doc/sphinx/reference/vcl_var.rst

```diff
@@ -393,6 +393,27 @@ req.hash_always_miss
 	This is useful to force-update the cache without invalidating
 	existing entries in case the fetch fails.
 
+req.uncacheable
+
+	Type: BOOL
+
+	Readable from: client
+
+	Writable from: client
```

Comment:
It should maybe be writable only in `vcl_recv` and `vcl_hash`, and become immutable once a lookup is initiated.